astropy/timeseries/downsample.py

```diff
@@ -10,7 +10,50 @@
 from astropy.timeseries.sampled import TimeSeries
 from astropy.utils.exceptions import AstropyUserWarning
 
-__all__ = ["aggregate_downsample"]
+__all__ = ["aggregate_downsample", "nanmean"]
```

Comment:
Do we want "nanmean"  to be in "public API"?

Reply:
@pllim The users might want to know which `aggregate_func` is being used. However, we could make `nanmean` private and include a clear explanation in the documentation where the `aggregate_func` parameter is described.

Reply:
I'll defer to @mhvk and @astrofrog  (or any maintainers who have preference on this).